versiondb/tsrocksdb/store.go

```diff
@@ -125,7 +125,7 @@ func (s Store) GetAtVersion(storeKey string, key []byte, version *int64) ([]byte
 // HasAtVersion implements VersionStore interface
 func (s Store) HasAtVersion(storeKey string, key []byte, version *int64) (bool, error) {
 	slice, err := s.GetAtVersionSlice(storeKey, key, version)
-	if err != nil {
+	if err != nil || slice == nil {
```

Comment:
it's a bad idea that I return the `nil` in the first place, let me see if I can return a empty one.